scripts/dev_scripts/integration_tests.sh

```diff
@@ -11,8 +11,27 @@ COMPARE_DEPS=$WORKSPACE/tests/dependency_analyzer/compare_dependencies.py
 COMPARE_JSON_OUT=$WORKSPACE/tests/e2e/compare_e2e_result.py
 TEST_REPO_FINDER=$WORKSPACE/tests/e2e/repo_finder/repo_finder.py
 RUN_MACARON="python -m macaron -o $WORKSPACE/output"
+PYTHON="python "
```

Comment:
Instead of `PYTHON` I think it's good to use a more descriptive variable name, like `CHECK_EXPECTED_OUTPUT`.